cmd/checkpoint/main_test.go

```diff
@@ -104,6 +104,22 @@ func TestProcess(t *testing.T) {
 			activeCheckpoints: map[string]*v1.Pod{"AA": {}},
 			localParents:      map[string]*v1.Pod{"AA": {}},
 		},
+		{
+			desc:                "Inactive pod-checkpointer, local parent, local running, api parent: should start",
```

Comment:
May be worth adding another test to make sure it still starts even with a local parent (the checkpointed copy always needs to run)